fryr/demo.html

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: Fryr Demo
+repository_slug: fryr
+js_download: true
+permalink: /fryr/demo
+tagline: Command window.location.hash like a cook in the kitchen
+logo: fryr.svg
+show_docs: true
+---
+<form class="wrapper">
+  <div class="input">
+    <label for="key">Key</label>
+    <input type="text" id="js-fryr-key" name="key" value="office_furniture" placeholder="Param value" />
+  </div>
+  <div class="input">
+    <label for="value">Value</label>
+    <input type="text" id="js-fryr-value" name="value" value="aquarium" placeholder="Param key" />
+  </div>
+
+  <section class="download">
+    <input type="submit" id="js-fryr-update" class="button" value="update()">
+    <a href="#" id="js-fryr-append" class="button">append()</a>
+    <a href="#" id="js-fryr-reset" class="button">Reset</a>
+  </section>
+</form>
+
+<pre id="js-params"></pre>
+
+<script type="text/javascript" src="./fryr.js"></script>
```

Comment:
@jamesmk So that's what I thought, however, GitHub has started serving raw files with the `text/plain` MIME type, and the browser won't execute these as scripts. 

The alternative, which I definitely think is the route to follow in the future but is a much more intensive PR, would be to auto-generate commits to this repo based on Webhooks in the other repos. For example, every time Fryr receives a new commit to master, the README rebuilds here and the local file is updated. This repo would never receive commits from humans.

Facebook appears to be doing this with [their React docs](https://github.com/facebook/react/tree/gh-pages), but I haven't had a chance to deep dive into "the how" of that system.